tutorials/monitoring/tutorial.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Monitoring NServiceBus solutions
+reviewed: 2017-10-02
+summary: Getting started with the monitoring capabilities built in to the Particular Platform.
+---
+
+This tutorial will guide you through getting started with the monitoring capabilities built in to the Particular Platform.
+
+You can follow along with this tutorial to configure any solution for monitoring with the Particular Platform. If you do not have a solution to monitor just yet, you can get started by downloading our sample solution.
+
+downloadbutton
+
+
+## Component overview
+
+The following diagram shows all of the components that are involved in monitoring an NServiceBus solution.
+
+```mermaid
+graph TD
+
+subgraph Endpoint1
+  metrics1(NServiceBus.Metrics) --> plugin1
+  plugin1(NServiceBus.Metrics.ServiceControl)
+end
+
+subgraph Endpoint2
+  metrics2(NServiceBus.Metrics) --> plugin2
+  plugin2(NServiceBus.Metrics.ServiceControl) 
+end
+
+plugin1 --> q(Monitoring Queue)
+plugin2 --> q(Monitoring Queue)
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+```
+
+![the above renders like this](component-overview.png)
+
+Each endpoint in the solution is configured to collect metrics about it's own operation using the NServiceBus.Metrics NuGet package. This package hooks into the NServiceBus pipeline and calculates key metric data. It also adds headers to outgoing messages to track queue length.
+
+The NServiceBus.Metrics.ServiceControl NuGet package collects the generated metric data and periodically sends it to a centralized monitoring queue. It does this with the same transport that the endpoints use to exchange business messages.
+
+A ServiceControl Monitoring Instance reads the messages from the monitoring queue and aggregates monitoring data for all of your endpoints. Each environment should have it's own Monitoring Instance. Monitoring Instances are created and managed with the ServiceControl Management Utility. 
```

Comment:
What is a "ServiceControl Monitoring Instance"? Is that something new? A special instance of ServiceControl? Do we need to preempt this question?

Reply:
It's a new thing. The direction we are heading is that "ServiceControl" is a suite of tools. One of those tools is a Monitoring Instance. We say something to this effect below in the section "Create Monitoring Instance". Here we are describing boxes on a diagram. Do we need to go into this greater depth yet?